querydsl-apt/src/main/java/com/querydsl/apt/VisitorConfig.java

```diff
@@ -13,8 +13,10 @@
  */
 package com.querydsl.apt;
 
+import com.querydsl.codegen.EntityType;
+
 /**
- * VisitorConfig defines the entity type specific visiting configuration
+ * {@code VisitorConfig} defines the {@link EntityType}-specific visiting configuration
```

Comment:
But according to the docs it is mostly useful when the link text is not code. At least the JDK javadocs use `@code` quite a lot for self references.